R/data_ops.R

```diff
@@ -32,11 +32,12 @@ bind_ems_data <- function(...) {
 #' @param param_code a character vector of parameter codes
 #' @param from_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
 #' @param to_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
+#' @param req_id A character vector of requisition id(s) of interest
```

Comment:
Could add a reference to `lt_lake_req()` - If you put it in square brackets roxygen will create a hyperlink to the man page for that function (e.g., "`See [lt_lakes_req()] for a list of long-term lakes monitoring sites requisition IDs`")